doc/nvim-treesitter.txt

```diff
@@ -165,7 +165,7 @@ get_node_text(node, bufnr)                        *ts_utils.get_node_text*
     return the text content of a node
 
 is_parent(dest, source)                           *ts_utils.is_parent*
-    determines wether `dest` is a parent of `source` 
+    determines wether `dest` is a parent of `source`
```

Comment:
whether ?

Reply:
That was already there, only the trailing space was removed, I can fix that typo here if that's ok

Reply:
As you prefer, yes we placed really a lot of typos in the docs :stuck_out_tongue: